test/extended/testdata/builds/volumes/buildconfig-s2i.yaml

```
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: List
+metadata: {}
+items:
+- apiVersion: build.openshift.io/v1
+  kind: BuildConfig
+  metadata:
+    name: mys2itest
+    labels:
+      name: test
+  spec:
+    triggers: []
+    runPolicy: Serial
+    source:
+      type: Git
+      git:
+        uri: 'https://github.com/sclorg/nodejs-ex'
+    strategy:
+      type: Source
+      sourceStrategy:
+        from:
+          kind: DockerImage
+          name: registry.ci.openshift.org/ocp/4.8:test-build-simples2i
+    resources: {}
+    postCommit: {}
+    nodeSelector: null
+  status:
+    lastVersion: 0
```

Status should be empty
```suggestion
  status: {}

```

---

updated in next push